static/bidder-params/criteo.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Criteo adapter params",
+    "description": "The schema to validate Criteo specific params accepted by Criteo adapter",
+    "type": "object",
+    "properties": {
+        "zoneid": {
+            "type": "number",
+            "description": "Impression's zone ID.",
+            "minimum": 0
+        },
+        "networkid": {
+            "type": "number",
+            "description": "Impression's network ID.",
+            "minimum": 0
+        }
+    },
+    "anyOf": [
+        {
+            "required": [
+                "zoneid"
+            ]
+        },
+        {
+            "required": [
+                "zoneId"
```

Comment:
I believe the content of a `required` field needs to map to a property, and there is no property with `zoneId`. Since this is a new adapter, is it necessary to provide publishers with a choice of capitalization?

Reply:
Indeed, I'll go with lower casing